src/services/completions.ts

```diff
@@ -1566,7 +1566,7 @@ namespace ts.Completions {
             // If you're in an interface you don't want to repeat things from super-interface. So just stop here.
             if (!isClassLike(decl)) return GlobalsSearch.Success;
 
-            const classElement = contextToken.parent;
+            const classElement = contextToken.kind === SyntaxKind.SemicolonToken ? contextToken.parent.parent : contextToken.parent;

```

Comment:
Is this the right fix? I get the sense that `contextToken.parent.parent` isn't actually a useful node, since the point of the check is simply to do nothing if you're not in the middle of the current class element.

Reply:
But the `contextToken.parent.parent` is a `ClassDeclaration` when `context.kind === SyntaxKind.SemicolonToken`. What I attempt to do is making the /\*1\*/ and /\*2\*/ should be in the same context.
```ts
class C {
	/*1*/ // `contextToken.parent` is `ClassDeclaration`
	private a = 1;
	/*2*/ // `contextToken.parent.parent` is `ClassDeclaration`
}
```
Do I miss something else?